modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenParameter.java

```diff
@@ -14,7 +14,7 @@
 
     public String example; // example value (x-example)
     public String jsonSchema;
-    public boolean isString, isInteger, isLong, isFloat, isDouble, isByteArray, isBinary, isBoolean, isDate, isDateTime;
+    public boolean isString, isInteger, isLong, isFloat, isDouble, isByteArray, isBinary, isBoolean, isDate, isDateTime, isUuid;
```

Comment:
Since this is a change in the very generic CodegenParameter, this will need the stamps from other collaborator since it can have effects on other generators.